moveit_ros/planning_interface/test/CMakeLists.txt

```diff
@@ -22,4 +22,15 @@ if (CATKIN_ENABLE_TESTING)
   add_rostest(python_move_group_ns.test)
   add_rostest(robot_state_update.test)
   add_rostest(cleanup.test)
+
+  add_library(moveit_planning_interface_test_serialize_msg_cpp_helper_python serialize_msg_python_cpp_helpers.cpp)
```

Comment:
What about introducing a variable, e.g. `HELPER_LIB`, for this monster of library name?